web-admin/src/features/projects/status/display-utils.ts

```diff
@@ -0,0 +1,31 @@
+import { V1ReconcileStatus } from "@rilldata/web-common/runtime-client";
+
+export function prettyReconcileStatus(status: V1ReconcileStatus) {
```

Comment:
Would javascript objects as maps be cleaner than these switch cases?

Reply:
Could definitely go either way on it. In these cases (small sets) I slightly prefer the simplicity & encapsulation of the `switch` statement.